tests/report_generator_test.py

```diff
@@ -0,0 +1,74 @@
+"""Report Generator Test"""
+
+import unittest
+
+from pipeline_dp.report_generator import ReportGenerator
+from pipeline_dp.dp_engine import AggregateParams, Metrics
+
+class ReportGeneratorTest(unittest.TestCase):
+  def test_report_empty(self):
+    ReportGenerator(None).report()  # no errors
+
+  def test_report_params(self):
+    params = AggregateParams(
+      max_partitions_contributed=2,
+      max_contributions_per_partition=1,
+      low=1,
+      high=5,
+      metrics=[Metrics.PRIVACY_ID_COUNT,Metrics.COUNT,Metrics.MEAN],
+    )
+    ReportGenerator(params).report() # no errors
+
+  def test_report_aggregate(self):
+    params1 = AggregateParams(
+      max_partitions_contributed=3,
+      max_contributions_per_partition=2,
+      low=1,
+      high=5,
+      metrics=[Metrics.PRIVACY_ID_COUNT,Metrics.COUNT, Metrics.MEAN],
+    )
+    params2 = AggregateParams(
+      max_partitions_contributed=1,
+      max_contributions_per_partition=3,
+      low=2,
+      high=10,
+      metrics=[Metrics.VAR,Metrics.SUM,Metrics.MEAN],
+      public_partitions = list(range(1,40)),
+    )
+    engine_stub = DPEngineStub()
```

Comment:
It looks like a test more for DPEngine than for ReportGenerator and that's a good to have such test. 
Could you please
 1. move it to dp_engine_tests?
 2.Use a DPEngine instead of DPEngineStub. It looks like DPEngine already have everything that's needed, so it's better to test it as well